Fix #17 user feed

wulymammoth opened the pull request:
There are two commits here:

1. Updates the README to correct the example for configuring authentication for Elixtagram
2. Fix the issue with `user_feed/2` as the old endpoint now returns a `404`

wulymammoth committed:
Fix README auth configuration example

wulymammoth committed:
Fix user_feed/2

- `users/self/feed` is now [`users/self/media/recent`](https://www.instagram.com/developer/endpoints/users/#get_users_media_recent_self)

wulymammoth committed:
Fix TravisCI build failure

- [reference](https://elixirforum.com/t/if-youre-using-travis-ci-you-probably-want-to-read-this/4915)
- [Hex.pm - Shared cert requirement (TLS 1.2+) updates being required by Fastly](https://status.fastly.com/incidents/4n5jrrhh5fyh)

sudostack commented:
Reverting to older commit and backing out changes to `travis.yml` based on latest updates. Discussion can be found here: https://elixirforum.com/t/if-youre-using-travis-ci-you-probably-want-to-read-this/4915

wulymammoth committed:
Cut changes to travis.yml

- Backing out the changes thanks to updates from TravisCI
- Discussion can be found [here](https://elixirforum.com/t/if-youre-using-travis-ci-you-probably-want-to-read-this/4915)

Zensavona commented:
Thanks @sudostack! This looks great. Merged this and I'll make a new release shortly.
